Tiltfile

```diff
@@ -676,6 +697,20 @@ if ci_tests:
         trigger_mode = trigger_mode,
         resource_deps = [], # testing/querysdk.sh handles waiting for query-server, not having deps gets the build earlier
     )
+    # launches tx-verifier binary and sets up monitoring script
+    k8s_resource(
+        "tx-verifier-with-monitor",
+        resource_deps = ["eth-devnet"],
+        labels = ["evm", "tx-verifier"],
```

Comment:
if you label this with two labels, it shows up twice in tilt. I'd probably only label these with `tx-verifier`.